src/secretsManager.ts

```diff
@@ -0,0 +1,36 @@
+import AWS from 'aws-sdk';
+
+const { IS_OFFLINE, AWS_REGION, STAGE } = process.env;
+export default class SecretsManager {
+    static initialize() {
+        if (IS_OFFLINE === 'true') {
+            AWS.config.update({
+                region: 'us-west-2',
+                accessKeyId: process.env.ACCESS_KEY,
+                secretAccessKey: process.env.SECRET_KEY,
+            });
+        }
+    }
+
+    static async getIntegrationTransformUrl(): Promise<string> {
+        const ssm = new AWS.SSM();
+        let path = process.env.INTEGRATION_TRANSFORM_PATH ?? 'fake_path';
```

Comment:
Avoid assigning values such as `fake_path`

Reply:
Yeah, I was debating what to put here. My thought was to either use an empty string or `fake_path`. I decided on `fake_path` because I felt it would make the error clearer when the error is printed out on this line.
https://github.com/awslabs/fhir-works-on-aws-deployment/blob/891ccdb259e44d9e9e83f03e4a52d67f0d7af7a3/src/secretsManager.ts#L29

I'm open to other ways of handling it though.

Reply:
Throwing the error here will allow lambda to terminate the error and return a 502 error. This code would be used during initialization of the FhirConfig